src/internal/storage/fileset/compaction.go

```diff
@@ -70,30 +38,22 @@ func indexOfCompacted(factor int64, inputs []*Primitive) int {
 	return l
 }
 
-// Compact compacts a set of filesets into an output fileset.
+// Compact writes the contents of ids to a new fileset with the specified ttl and returns the ID
+// Compact always returns the ID of a primitive fileset.
 func (s *Storage) Compact(ctx context.Context, ids []ID, ttl time.Duration, opts ...index.Option) (*ID, error) {
```

Comment:
Where did the level based compaction go?

Reply:
Okay I see it down below. So there is no way to perform level based compaction without the distributed compactor?

Reply:
Yeah.